### PR TITLE
replace data-uri placeholder

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Fixed validation error. @kul3r4 PTAL

File: src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html [hunk 1]
```diff
@@ -42,15 +42,17 @@
   <!--
     ...and interactive content embedded via iframes.
 
-    One thing is important to be aware of: the AMP runtime will only show iframes located at the top of a website if they provide a placeholder image. The charts in the sample are updated in real-time which means we cannot provide a meaningful placeholder. As a workaround we use an empty image as placeholder, which is directly embedded via an [data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/data_URIs).
+    One thing is important to be aware of: the AMP runtime will only show iframes located at the top of a website if they provide a placeholder image. The charts in the sample are updated in real-time which means we cannot provide a placeholder image. Instead, we use an image of the chart background which stretches well to different widths.
```
Comment (kul3r4) [inline src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html]: Suggestion:
One thing is important to be aware of:-> One thing to be aware of: 
or
It is important to be aware that the AMP runtime...

**Comment by kul3r4** [comment]
Just one suggestion on the narrative, always LGTM

**Comment by sebastianbenz** [comment]
Thanks - fixed the wording.

**Pull request merged**